blog/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+
+class Account(models.Model):
+    user = models.OneToOneField(User, related_name='account', on_delete=models.CASCADE)
```

Comment:
In my view having one to one relationship will only allow you have just one account per user.
```suggestion
    user = models.OneToOneField(User, related_name='account', on_delete=models.CASCADE) 
```

Reply:
Am still for OneToOneField option here because I am trying to create a relationship between one user and the one profile. Am ready to here your views on this

Reply:
What about the other accounts @sangkips? How will we be able to connect the user to the other profile accounts they can have?

Reply:
@Eugene-Kwaka The reason why I am saying one user should have a single profile/account is that I am trying to imagine if you want to update your profile/account, how will you tell which profile/account you want your changes to go to?

Reply:
Every Account has its unique id. So you will "direct" changes/updates to the specific Account.